tests/test_qubit_device.py

```diff
@@ -1426,7 +1426,7 @@ def test_tracker_multi_execution(self, dev_name):
         ):
             assert tracked_r == expected_r
 
-    @pytest.mark.all_interfaces
+    @pytest.mark.autograd
```

Comment:
Is this just to save resources?

Reply:
think just something I noticed while going through the tests.  This is an autograd test, not a all interfaces test.